Index: src/interfaces/INETH.sol
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>// SPDX-License-Identifier: GPL-3.0\npragma solidity ^0.8.7;\nimport \"openzeppelin-contracts/token/ERC20/IERC20.sol\";\n\ninterface INETH is IERC20{\n    function whiteListMint(uint256 amount, address account) external;\n    function whiteListBurn(uint256 amount, address account) external;\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/interfaces/INETH.sol b/src/interfaces/INETH.sol
--- a/src/interfaces/INETH.sol	(revision 462c58f1f42f846e46f44bf877392d9d790ab0b2)
+++ b/src/interfaces/INETH.sol	(date 1673425895529)
@@ -1,8 +1,9 @@
 // SPDX-License-Identifier: GPL-3.0
 pragma solidity ^0.8.7;
+
 import "openzeppelin-contracts/token/ERC20/IERC20.sol";
 
-interface INETH is IERC20{
+interface INETH is IERC20 {
     function whiteListMint(uint256 amount, address account) external;
     function whiteListBurn(uint256 amount, address account) external;
 }
